src/meili/traffic_segment_matcher.cc

```diff
@@ -82,6 +84,22 @@ namespace {
     return merged;
   }
 
+
+  //TODO: remove this when debugging phase is finally over
```

Comment:
this is very very useful for debugging. gdb in tandem with this is very nice when working with longer traces. notice its commented out below but until im sure all the bugs are sufficiently gone or have time to make better tests to expose these (lots of TODOS already in the code there) this is the next best thing.